specification/security/resource-manager/Microsoft.Security/preview/2021-08-01-preview/antiMalwareSettings.json

```diff
@@ -0,0 +1,171 @@
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{resourceId}/providers/Microsoft.Security/antiMalwareSettings/{settingName}": {
+      "get": {
+        "x-ms-examples": {
+          "Gets the anti malware settings for the specified resource.": {
+            "$ref": "./examples/AntiMalware/GetAntiMalwareSettings_example.json"
+          }
+        },
+        "tags": [
+          "AntiMalware"
+        ],
+        "description": "Gets the Anti Malware settings for the specified resource.",
+        "operationId": "AntiMalware_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceId"
+          },
+          {
+            "$ref": "#/parameters/AntiMalwareSettingName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get Anti Malware settings.",
+            "schema": {
+              "$ref": "#/definitions/AntiMalwareSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Creates or updates the Anti Malware settings on a specified resource.": {
+            "$ref": "./examples/AntiMalware/PutAntiMalwareSettings_example.json"
+          }
+        },
+        "tags": [
+          "AntiMalware"
+        ],
+        "description": "Creates or updates the Anti Malware settings on a specified resource.",
+        "operationId": "AntiMalware_Create",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceId"
+          },
+          {
+            "$ref": "#/parameters/AntiMalwareSettingName"
+          },
+          {
+            "$ref": "#/parameters/AntiMalwareSetting"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to put Anti Malware settings.",
+            "schema": {
+              "$ref": "#/definitions/AntiMalwareSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AntiMalwareProperties": {
+      "type": "object",
+      "properties": {
+        "isEnabled": {
```

Comment:
>"isEnabled": { [](http://example.com/codeflow?start=8&length=14)

Please avoid using a boolean type. Instead model it as a string enum. e.g. "status": ["enabled", "disabled"]. It is more flexible and additional values can be added later without a breaking change.

[https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review)